src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialFormView.tsx

```diff
@@ -118,39 +125,48 @@ const MaterialFormView: React.FC<MaterialFormViewProps> = ({
             />
           </FormControl>
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={12}>
```

Comment:
this is kind of a nitpick but in a small screen view, the add button remains the same fixed width while the autocomplete shrinks. you could separate them into separate grid items each with xs of 6 (take up half a row each so that they remain equal size no matter the window size

Reply:
it'll still look the exact same as you have it now in fullscreen, just the width of both components will be equal to each other